### ISSUE TITLE
Add arm64 builds for minecraft-ecsfargate-watchdog

### ISSUE THREAD

**Issue by Stealthii** [doctorray117/minecraft-ondemand#27]
Now that [AWS supports arm64 on ECS Fargate](https://aws.amazon.com/about-aws/whats-new/2021/11/aws-fargate-amazon-ecs-aws-graviton2-processors/), it would be nice to push builds of the watchdog container to support that.

As of #6 the image needs no additional changes, and I have a test image up at `stealthii/minecraft-ecsfargate-watchdog`.

Here's the line I used to build:
```sh
docker buildx build --platform=linux/arm64 --tag stealthii/minecraft-ecsfargate-watchdog --push minecraft-ecsfargate-watchdog 
```

I've been using arm64 successfully on the Lambda tasks, the watchdog container, and the 1.18.x Fabric server running under this project's configuration!

**Comment by doctorray117** [comment]
Good idea!  I'll check out your stuff probably over the weekend.  hard to grab time otherwise.  Are you using the itzg/minecraft-server image for arm for this?

**Comment by Stealthii** [comment]
I am! Fargate Spot unfortunately isn't supported, the price sits in the middle between amd64 spot/non-spot. On a low-pop server I've noticed comparable tick performance, and a noticable memory usage drop. Worth doing your own testing before you recommend it over amd64.

Annoyingly you can only configure arm64 on the new ECS interface, and currently you cannot set empty variables in the new interface, making configuration a nightmare (I've found using ECS memory reservations more reliable when coupled with itzg's options for this: `-e MEMORY="" -e JVM_XX_OPTS="-XX:MaxRAMPercentage=75"` + container memory limits)
